<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageInfo</class>
 <widget class="QWidget" name="MessageInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>65</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>65</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>65</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Image_Lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../image/Image.qrc">:/new/user/user/touxiang02.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="userName_Lab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="userMessage_Lab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>35</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer_Lab">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
